src/l500/l535-options.cpp

```diff
@@ -0,0 +1,74 @@
+//// License: Apache 2.0. See LICENSE file in root directory.
+//// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "l535-options.h"
+#include "l535-amc-option.h"
+#include "l535-preset-option.h"
+#include "l500-private.h"
+#include "l500-depth.h"
+
+namespace librealsense
```

Comment:
I don't like the .cpp files indenting like this.
Won't a single `using librealsense::ivcam2::l535::device_options;` do?

Reply:
done